c/src/error.rs

```diff
@@ -68,6 +73,14 @@ pub(super) fn try_release_optional_string(result: Option<Result<String>>) -> *mu
     ok_record_flatten(result).map(release_string).unwrap_or_else(null_mut)
 }
 
+pub(super) fn try_unwrap_arc<T>(result: Result<Arc<T>>) -> *const T {
+    try_unwrap_arc_optional(ok_record(result))
+}
+
+pub(super) fn try_unwrap_arc_optional<T>(result: Option<Arc<T>>) -> *const T {
+    result.map(unwrap_arc).unwrap_or_else(null)
+}
```

Comment:
These should be `try_release_arc` and `try_release_optional_arc` (see my earlier comment)

Reply:
Done.